Added 'language' field to select which wikipedia will be searched

Witia1 opened the pull request:
<!-- Thank you for contributing to the Primerpedia project! -->
<!-- If applicable, please use the following URL template to provide a preview of the changes: -->

➡ [**Live preview of the changes**](http://rawgit.com/{USERNAME}/primerpedia/{BRANCH-NAME}/index.html)

Witia1 committed:
Added 'language' field to select which wikipedia will be searched (now only biggest)

waldyrious commented on .gitignore:
```diff
@@ -0,0 +1 @@
+/.brackets.json
```
Please remove this, since it will only apply to people who use this specific editor.

Note that you can add local-only ignore rules to the `.git/info/exclude` file within a given repository, so that will probably be helpful :)

Witia1 replied:
I removed .gitignore.

waldyrious commented:
Thanks for the pull request! I will try to review this later today and will give you feedback, if needed.

Witia1 commented:
I see in this pull request all my today commits. At this moment I want to request only https://github.com/waldyrious/primerpedia/pull/60/commits/937c7e3ebc907cc75cdf224ce68a1c52e47d1a37. Next commit is not yet ready to pull request. How can I do it and request separetly later? Sorry for the confusion :)

waldyrious committed:
Various changes

- change language input field to use a placeholder instead of default value (and to not be required)
- moved sans-serif definition to the body rather than #content

waldyrious commented:
@Witia1 I made some changes to the implementation, as we discussed offline, and tested the implementation locally. Works great, nice work! :+1:
